plugins/com.google.cloud.tools.eclipse.appengine.validation/plugin.properties

```diff
@@ -5,3 +5,4 @@ appengine.blacklist.marker=App Engine Configuration Problem
 web.xml=Web XML Validator
 servlet.marker=Servlet Marker
 maven.plugin.marker=Obsolete Maven App Engine Plugin
+undefined.servlet.marker=Undefined Servlet Class
```

Comment:
This message appears as the error type in the problems view. For a similar error ("_____ cannot be resolved to a type") Eclipse gives the type as "Java Problem." But I'm fine to include the class name if you think it would be more descriptive